lib/js-api/src/api/gs-core-rest-api/model/hold.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Hold {
+    id: string;
+    name: string;
+    reason?: string;
```

Comment:
Should reason be optional? I think it's required in API definition of `HoldModelEntry`

Reply:
It depends where we use it.
When we are getting hold it can be only name and id, to save hold we need name and reason.

Reply:
So there is no option to get the reason after hold is created?

Reply:
after conversation was decided to leave this interface as it is